backend/capellacollab/projects/capellamodels/models.py

```diff
@@ -76,8 +84,12 @@ class DatabaseCapellaModel(Base):
 
     editing_mode = Column(Enum(EditingMode))
 
-    t4c_models = relationship("DatabaseT4CModel", back_populates="model")
-    git_models = relationship("DB_GitModel", back_populates="model")
+    t4c_models: t.Sequence[DatabaseT4CModel] = relationship(
```

Comment:
I cannot find any documentation on the type returned by the relationship, how do you know it’s a `Sequence`?

Reply:
I finally found that it’s an [InstrumentedList](https://docs.sqlalchemy.org/en/20/orm/collection_api.html#sqlalchemy.orm.collections.InstrumentedList), which inherits from the list.
I don’t know what to do about this information tho 🙊

Reply:
A sequence is a primitive type for a list, so that should be fine. We do not need the complete list interface so far.